plugin.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "com.mattermost.aws-sns",
+    "name": "AWS SNS",
+    "description": "This plugin receives SNS notification from Alerts created by AWS Cloudwatch and sent via AWS SNS.",
+    "version": "0.1.1",
+    "server": {
+        "executables": {
+            "linux-amd64": "server/dist/plugin-linux-amd64",
+            "darwin-amd64": "server/dist/plugin-darwin-amd64",
+            "windows-amd64": "server/dist/plugin-windows-amd64.exe"
+        }
+    },
+    "settings_schema": {
+        "header": "",
+        "footer": "",
+        "settings": [{
+            "key": "TeamChannel",
+            "display_name": "Team/Channel",
+            "type": "text",
+            "help_text": "Name of the Team and Channel you want to create/use to receive the SNS messages. Split by comma. ie. teamA,channelXYZ"
```

Comment:
this bugs me a little, is there any reason to not have two config fields?

Reply:
There is no previous convention for specifying team and channel, though the autolink plugin does something similar https://github.com/mattermost/mattermost-plugin-autolink/pull/43/files#diff-e1fc94e27d16594718ce6393a0adadb5R66

There are other possible approaches for specifying where alerts are subscribed to. Splitting it into two separate settings for Team and Channel (with an autocomplete field) is one.

Another may be copying the URL of the channel they want to send notifications to into this field.